tests/test_stac.py

```diff
@@ -3,6 +3,20 @@
 import pytest
 
 from stactools.ecmwf_forecast import stac
+import json
+import os
+import urllib.request
+
+blob_file = ("https://ai4edataeuwest.blob.core.windows.net/ecmwf/20231019/00z/"
+         "0p4-beta/wave/20231019000000-0h-wave-fc.grib2")
+
+local_files = ["20231019000000-0h-wave-fc.grib2",
+               "20231019/00z/0p4-beta/wave/20231019000000-0h-wave-fc.grib2"]
+for i in range(len(local_files)):
+    if not os.path.exists(local_files[i]):
+        if i==1:
+            os.makedirs(local_files[i].split('.')[0], exist_ok=True)
```

Comment:
```suggestion
        os.makedirs(local_file.split('.')[0], exist_ok=True)
```

maybe easier to just always try creating the directory.

Reply:
I only want to create a directory for the longer filepath therefore I kept as is.